ext/net/lib.deno_net.d.ts

```diff
@@ -49,7 +49,12 @@ declare namespace Deno {
     /** The port to listen on. */
     port: number;
     /** A literal IP address or host name that can be resolved to an IP address.
-     * If not specified, defaults to `0.0.0.0`. */
+     * If not specified, defaults to `0.0.0.0`.
+     *
+     * __Note about 0.0.0.0__ While listeninig 0.0.0.0 works on all platforms,
+     * the browsers on Windows don't work with the address 0.0.0.0.
+     * You should show the message like `server running on localhost:8080` instead of
+     * `server running on 0.0.0.0:8080` if your program supports Windows. */
```

Comment:
_Consider_ following the precedent set by the existing comment and backtick fence `0.0.0.0` occurrences:

```suggestion
     * __Note about `0.0.0.0`__ While listeninig `0.0.0.0` works on all platforms,
     * the browsers on Windows don't work with the address `0.0.0.0`.
     * You should show the message like `server running on localhost:8080` instead of
     * `server running on 0.0.0.0:8080` if your program supports Windows. */
```

Reply:
Makes sense. Thanks!